### PR TITLE
feat(@desktop): Replace image with animation on the splash screen

### LINKED ISSUE #4984: Update splash screen asset
### Description
Splash screen logo should be animated, currently a static logo is used. Use either lottie file from [here](https://lottiefiles.com/share/xl7pjmjz) or a gif animated image.

### PR DESCRIPTION
**Pull request by MishkaRogachev**
Close #4984

### What does the PR do

Replace image with animation on the splash screen

### Affected areas

Spash screen

### Screenshot of functionality

gif_with_text-2022-05-12_19 11 18

### CODE REVIEW

**Comment by MishkaRogachev** [comment]
It seems that LottieAnimation in Qt does not work properly, should i use gif instead?

**Comment by iurimatias** [comment]
is this WIP?

https://user-images.githubusercontent.com/176720/168119860-2a76f0f0-c6c6-42ae-a7fa-525634b519a2.mov

**Comment by MishkaRogachev** [comment]
> is this WIP?
> 
>  tes.mov

6 min ago was it Draft, but now you can check again ;)

**Review by iurimatias — CHANGES_REQUESTED** [review]
it's blank for me

**Comment by MishkaRogachev** [comment]
> it's blank for me

That's strange. Maybee some issue wirh AnimationImage on MAC. I asked team to check it: https://discordapp.com/channels/@me/972849023155318824/974573458136641556

**Comment by endulab** [comment]
Works for me. 

https://user-images.githubusercontent.com/61889657/168235639-22a22a5e-a9cf-4de3-8143-68a230fe9298.mov

File: ui/app/mainui/SplashScreen.qml [hunk 1]
```diff
@@ -17,24 +17,17 @@ Item {
...
-//        loops: LottieAnimation.Infinite
-//        quality: LottieAnimation.MediumQuality
-//        source: Style.lottie("status_splash")
-//        onStatusChanged: {
-//            if (status === LottieAnimation.Ready) {
-//                start();
-//            }
-//        }
-//    }
+    // LottieAnimation {
```
Comment (endulab) [inline ui/app/mainui/SplashScreen.qml:26]: Do we want to keep this ?
If yes, then it would be good to add the comment why it is here.

**Comment by alexandraB99** [comment]
works for me, however the circle edges are a bit sharp black, like it has a border

**Comment by MishkaRogachev** [comment]
> works for me, however the circle edges are a bit sharp black, like it has a border

I used gif converter from assest web app. IMHO we should decide how to deal with animations, probably when we move to Qt6 lottie will work properly.

**Comment by MishkaRogachev** [comment]
> Everything works for me. Except I would suggest to use `scale: 0.5` on `AnimationImage` to support high-res displays. This would also require a larger original gif. But it looks much better with it (I didn't have a larger gif though)
> 
> P.S. I've just finished fixing blurry emojis, so now I have an eye on it smile
> 
> ## scale=1
>  2022-05-13.19.09.48.mov 
> ## scale=2
>  2022-05-13.19.10.34.mov

Nice idea!

**Review by iurimatias — APPROVED** [review]

**Pull request merged**